lib/charms/observability_libs/v1/kubernetes_service_patch.py

```diff
@@ -187,6 +206,7 @@ def __init__(
         super().__init__(charm, "kubernetes-service-patch")
         self.charm = charm
         self.service_name = service_name if service_name else self._app
```

Comment:
If we already decide on the service name ourselves, might as well append `-lb` to avoid the error below.

```suggestion
        self.service_name = service_name if service_name else self._app if service_type == "LoadBalancer" else f"{self._app}-lb"
```

Reply:
done but not as condensed just for readability